apps/site/lib/site_web/templates/schedule/_line_header.html.eex

```diff
@@ -1,5 +1,5 @@
 <div class="schedule__header <%= header_class(@route) %>">
-  <div class="schedule__header-container">
+  <div class="schedule__header-container notranslate">
```

Comment:
Can we undo this? We haven't committed to preventing translation of route names yet. This spot also prevents translation of the tab names (schedules & info, alerts) which I don't think we want.